app/web/src/newhotness/Explore.vue

```diff
@@ -1608,7 +1647,7 @@ onBeforeUnmount(() => {
   removeEmitters();
   mouseEmitter.off("click", onClick);
 });
-watch([router.currentRoute], setSelectionsFromQuery);
+// watch([router.currentRoute], setSelectionsFromQuery);
```

Comment:
Removing this breaks retainSessionState, since when it loads up it just pushes values back to the url. In general, this enforces that we react to route changes that don't unmount this component, so we can let the URL control the page